<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QWidget" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>x width</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="VisivblePeriod">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>10</second>
        <year>1752</year>
        <month>9</month>
        <day>14</day>
       </datetime>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>10</second>
       </time>
      </property>
      <property name="minimumTime">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>5</second>
       </time>
      </property>
      <property name="displayFormat">
       <string>H:mm:ss</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="UpdateTime">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string> H:mm:ss:zzz</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCustomPlot" name="custPlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>77</horstretch>
       <verstretch>88</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>custPlot</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
